css/style.css

```diff
@@ -9,3 +9,126 @@ header {
   background: url("../images/header_bg.jpg") no-repeat center top;
   background-size: cover;
 }
+
+body {
+	background-color: #f0efef;
+	font-family: 'Open Sans',sans-serif;
+	margin: 0 auto;
+}
+
+h1,h2,p {
+	color: #FFFFFF;
+}
+
+h1 {
+	padding-top: 20px;
+}
+
+h1,ul,p.b1 {
+	padding-left: 20px;
+}
+
+h2 {
+ 	font-weight; 300;
+ 	text-align: left;
+ 	padding-top: 200px;
+ 	padding-bottom: 50px;
+}
+
+h3 {
+	font-size: 20px;
+	font-weight: 400;
+	color: #121212;
+	padding-left: 20px;
+	padding-top: 40px;
```

Comment:
You are using this to pad the top of the benefits. However, if you remove it, you can add this and pad the bottom by applying the padding to your .Benefits class.